vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBox.java

```diff
@@ -385,30 +394,6 @@ public void setRenderer(Renderer<T> renderer) {
         scheduleRender();
     }
 
-    /**
-     * {@inheritDoc}
-     * <p>
-     * Filtering will use a case insensitive match to show all items where the
-     * filter text is a substring of the label displayed for that item, which
-     * you can configure with
-     * {@link #setItemLabelGenerator(ItemLabelGenerator)}.
-     * <p>
-     * Filtering will be handled in the client-side if the size of the data set
-     * is less than the page size. To force client-side filtering with a larger
-     * data set (at the cost of increased network traffic), you can increase the
-     * page size with {@link #setPageSize(int)}.
-     * <p>
-     * Setting the items creates a new DataProvider, which in turn resets the
-     * combo box's value to {@code null}. If you want to add and remove items to
-     * the current item set without resetting the value, you should update the
-     * previously set item collection and call
-     * {@code getDataProvider().refreshAll()}.
-     */
-    @Override
-    public void setItems(Collection<T> items) {
```

Comment:
I think you should retain overriding this method by defer to super implementation because the javadocs explain how this method works with CB, while the super method does not

Reply:
Done